app/controllers/metrics_controller.rb

```diff
@@ -12,5 +12,7 @@ def show
     @summary = SingleContentItemPresenter
       .parse_metrics(service.fetch(service_params))
       .parse_time_series(service.fetch_time_series(service_params))
+  rescue GdsApi::HTTPNotFound
```

Comment:
What about being declarative about the exceptions with [rescue_from](https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from) in this controller?. It decreases the size of the action and let the reader focus on what the action does.